src/components/ItemCreateForm/ItemCreateForm.tsx

```diff
@@ -11,7 +11,7 @@ const NAME_MAX_LENGTH = 50;
 const VALIDATION_MSG_NAME =
   'Name is required, cannot contain special characters and must be less than 50 characters in length.';
 const nameIsValid = (name: string): boolean => {
```

Comment:
this could be shortened to...
```tsx
cost REGEX = new Regex(`^[\w ]{${NAME_MIN_LENGTH},${NAME_MAX_LENGTH}}$`);
const nameIsValid = (name: string): boolean => {
  return REGEX.test(name);
}
```

Reply:
if you don't want underscore, you probably want the following:

```tsx
const REGEX = new Regex(`^[a-zA-Z0-9 ]{${NAME_MIN_LENGTH},${NAME_MAX_LENGTH}}$`);
const nameIsValid = (name: string): boolean => {
  return REGEX.test(name);
}
```

Reply:
I just added just did [\w_]

Reply:
[\\w_]

Reply:
have you pushed this yet?